spec.html

```diff
@@ -28211,6 +28211,7 @@ <h1>
             1. For each ExportEntry Record _e_ of _module_.[[IndirectExportEntries]], do
               1. If _e_.[[ExportName]] is _exportName_, then
                 1. Assert: _e_.[[ModuleRequest]] is not *null*.
+                1. Assert: _e_.[[ImportName]] is neither ~all-but-default~ nor *null*.
```

Comment:
Instead of ruling out two of the four possible states here, and then switching between the remaining two later, I would add another assert in the "else" case below (after the existing "imports a specific binding" assert):
```
                  1. Assert: _e_.[[ImportName]] is a String.
```
That seems clearer.